modules/applications/pages/download-and-launch.adoc

```diff
@@ -175,7 +180,7 @@ Run the following command to launch Bonita UI Builder:
 docker compose up -d
 ----
 
-Then, you can access Bonita UI Builder at `http://localhost`.
+Then, you can access Bonita UI Builder at `http://localhost:8090`.
```

Comment:
In the demo, the reverse proxy was handling the redirection to the proper port. Did it changed ?

Reply:
When I tested it, it didn't redirect me properly to the right URL, hence this change

Reply:
OK...it just worked now 🤔